extensions/wikia/ArticleAsJson/ArticleAsJson.class.php

```diff
@@ -8,7 +8,6 @@ class ArticleAsJson extends WikiaService {
 	];
 
 	const CACHE_VERSION = '0.0.3';
-	const CACHE_VERSION_FOR_SEO_FRIENDLY_IMAGES = 1; // for $wgEnableSeoFriendlyImagesForMobile
```

Comment:
the cache version is going to change. The code that was enabled for few weeks was using the Cache version 1. Now it's going to use version 0.0.3. The question is if it should change to 0.0.3 or maybe we should leave 1. I'm not sure if it's going to break something or not.